shell/platform/windows/system_utils_unittests.cc

```diff
@@ -12,38 +12,6 @@
 namespace flutter {
 namespace testing {
 
-class MockWindowsRegistry : public WindowsRegistry {
```

Comment:
The `WindowsRegistry` class is now unused. I am unsure if we should remove it altogether or not.

Reply:
Sadly, yes please remove this. We can use git's history to resurrect this code when we need the registry again.